packages/@sanity/cli/src/cli.ts

```diff
@@ -130,14 +130,14 @@ export async function runCli(cliRoot: string, {cliVersion}: {cliVersion: string}
   const cliCommandTrace = telemetry.trace(CliCommand, {
     groupOrCommand: args.groupOrCommand,
     extraArguments: args.extraArguments,
-    commandArguments: args.argsWithoutOptions,
+    commandArguments: args.argsWithoutOptions.slice(0, 10),
   })
 
   cliCommandTrace.start()
   cliCommandTrace.log({
```

Comment:
@svirs This is basically redundant now that we pass (most of) this as trace context. Would it be ok for us to remove explicitly logging this? The only thing we don't currently include in the context is `coreOptions`, but we can move it over so we have it there too.

Reply:
That makes sense as long as we capture that in the context

Reply:
Cool - I've removed the redundant logging in 0825cac3b2063d30d0e94b0c253fdac99c608ef9